x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -2804,7 +2817,7 @@ export class RulesClient {
         });
       }
     });
-    return { errors, taskIdsToEnable };
+    return { errors, rules, taskIdsToEnable };
```

Comment:
I'm looking at the `bulkEditOcc` function, and it seems like we're also returning all of the `result.saved_objects` as well, alongside all of the successfully updated rules. Do we want to follow that pattern here as well?

Reply:
@JiaweiWu We are excluding errors as well in bulkEdit, but in `x-pack/plugins/alerting/server/rules_client/lib/retry_if_bulk_edit_conflicts.ts` function.

`const results = [...accResults, ...resultSavedObjects.filter((res) => res.error === undefined)];`